library/hdfs-library/src/main/java/uk/gov/gchq/gaffer/hdfs/operation/reducer/GafferReducer.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 Crown Copyright
```

Comment:
Copyright header should be updated - same goes for the rest of the new files in this PR.